### ISSUE TITLE
Unable to run dmx-hue and Resolume in parallel

### ISSUE THREAD

**Issue by toxic-teddy** [sinedied/dmx-hue#32]
Hi.

I'm trying to use Artnet connections set up over 2 software to control 2 light sources.

The first source is my Philips Hue lights through QLC+ using DMX Hue.

The second source is a pixel LED strip connected to an Artnet controller (SP801E). I am controlling the pixel strip using Resolume.

I am able to control the pixel strips and the Philips Hue lights independently using either software, but when I'm trying to run them together, I'm running into issues.

If I run Resolume first, then trying to run dmx-hue gives me the error:
"Error: bind EADDRINUSE 0.0.0.0:6454"

Conversely, if I run dmx-hue first (it runs successfully), and then I try running Resolume, I get this error in the Resolume DMX settings:
"Uh-oh. Art-Net can't be started. Another program is already using the port."

What would be the best way to get both running at the same time?

**Comment by sinedied** [comment]
Is it possible to disable the Art-Net node on Resolume (it's not needed for output), or change the Art-Net port on Resolume and pixel strip?

Another option would be to use a Virtual Machine or Docker container to run DMX Hue, with a bridged network, to allow both software to listen on the same port.

**Comment by rjmunro** [comment]
I was able to get dmx-hue to run on the same machine as ChamSys MagicQ by changing the way the Art Net socket is created so that it can be shared by multiple apps. This will probably work for other apps. It would be good if you could try my branch with Resolume.

Download or clone my branch from the PR to a folder, then from that folder you can run:

```bash
npm install
bin/dmx-hue
```

To try it.
